modules/xsimd/8.1.0/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
```

Comment:
nit: If you want to also verify against the upcoming Bazel 8, you can add `8.0.0rc2` here. (8.x doesn't work yet)

Reply:
Got it